### PR TITLE
add new recipe selenzy_wrapper v0.0.4

### PR DESCRIPTION
**Pull request by kenza12**
Describe your pull request here

Add recipe for **selenzy_wrapper** 0.0.4: Python wrapper of selenzy tool

https://github.com/brsynth/selenzy-wrapper/
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by telatin** [comment]
Hello @kenza12, are you sure you need to restrict biopython to =1.77 only?

**Comment by kenza12** [comment]
Hello @telatin , I just update the recipe and added a new restriction for **biopython >1.70,<=1.77** , I also added some tests but it failed for memory issues. Is there a limit for memory usage in bioconda ? If yes, is it possible to increase it ? Thanks for your work.

**Comment by telatin** [comment]
> Hello @telatin , I just update the recipe and added a new restriction for **biopython >1.70,<=1.77** , I also added some tests but it failed for memory issues. Is there a limit for memory usage in bioconda ? If yes, is it possible to increase it ? Thanks for your work.

Not that I'm aware, I usually put the more extensive tests on the repository CI and leave the bare minimum in conda to just check that the program runs / has the correct version. Is that a possibility?

**Comment by kenza12** [comment]
> > Hello @telatin , I just update the recipe and added a new restriction for **biopython >1.70,<=1.77** , I also added some tests but it failed for memory issues. Is there a limit for memory usage in bioconda ? If yes, is it possible to increase it ? Thanks for your work.
> 
> Not that I'm aware, I usually put the more extensive tests on the repository CI and leave the bare minimum in conda to just check that the program runs / has the correct version. Is that a possibility?

Hi @telatin , 

Thanks for your feedback. I don’t think it’s a problem about checking the program version. It doesn’t need so much resources to do that.

And I got this **error** in the **test session** :

```
17:59:59 BIOCONDA INFO (OUT) OSError: [Errno 12] Cannot allocate memory
18:00:13 BIOCONDA ERROR COMMAND FAILED (exited with 1): docker run -t --net host --rm -v /tmp/tmpnu7niyow/build_script.bash:/opt/build_script.bash -v /home/circleci/project/miniconda/conda-bld/:/opt/host-conda-bld -v /home/circleci/project/recipes/selenzy_wrapper:/opt/recipe -e LANG=en_US.UTF-8 -e HOST_USER_ID=1001 quay.io/bioconda/bioconda-utils-build-env-cos7:0.17.8 /bin/bash /opt/build_script.bash

18:00:13 BIOCONDA ERROR BUILD FAILED recipes/selenzy_wrapper
18:00:13 BIOCONDA INFO (COMMAND) conda build purge
18:00:14 BIOCONDA ERROR BUILD SUMMARY: of 1 recipes, 1 failed and 0 were skipped. Details of recipes and environments follow.
18:00:14 BIOCONDA ERROR BUILD SUMMARY: FAILED recipe recipes/selenzy_wrapper

Exited with code exit status 1
```

I see that bioconda use **Linux Medium** resources (**2 CPU, 7.5 GB RAM**), is that set automatically ? I don’t know if it needs to be rerun with larger resources maybe ? I’m a novice in bioconda and all CircleCI tests.

Thanks for your help.

Best wishes.

**Comment by kenza12** [comment]
Hi @telatin , @bioconda/core ,

Do you have any update about my issue please ?

Thanks for your time.

File: recipes/selenzy_wrapper/run_test.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+echo -e "\n\n*** TEST ***\n\n"
+python -m selenzy_wrapper --help
+pytest -v
```
[the remark this answers is not in the document]
Comment (dpryan79) [reply recipes/selenzy_wrapper/run_test.sh:5]: Also, is there no executable installed for this? Typically python packages would be executed like `selenzy --help` or something like that.

[the remark this answers is not in the document]
Comment (kenza12) [reply recipes/selenzy_wrapper/run_test.sh]: Hi @dpryan79 ,

Thanks for your reply.

I tried to run only one test but it always returns the same memory error. Finally, we decided not to run tests and to only check the imports and the help command line: `python -m selenzy_wrapper --help` .
We don't have an executable  `selenzy --help` , we execute the package with this command `python -m selenzy_wrapper --help` .

Thank your for your work.

**Review by telatin — APPROVED** [review]
Tests issue bypassed

File: recipes/selenzy_wrapper/run_test.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (telatin) [reply recipes/selenzy_wrapper/run_test.sh]: Maybe in a future release adding a script to be invoked will make the tool more handy :)
